src/std/hashmap.md

```diff
@@ -24,3 +24,21 @@ fn main() {
     }
 }
 ```
+
+
+<details>
+
+* The HashMap is not defined in the prelude and needs to be brought into scope.
+* In a hashmap, types with the `Copy` trait (`i32`) are copied into the hashmap, and owned values (`String`) are moved into the hashmap.
```

Comment:
There's not actually any special-casing for `Copy` types is there?

Reply:
I guess not, but I thought this would be a common question. Should I remove it?

Reply:
I'm just not sure what you are trying to say here. Doesn't `HashMap` do exactly the same thing for both `i32` and `String`?